### PR TITLE
secure login gov

### PR DESCRIPTION
**Pull request by Mits1388**
add dependencies
add suit for secure login gov
add test to validate login form with empty date
add test to validate login form with any email
add test to validate login form with any password
add test to validate login form with any data
refactor pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <artifactId>demo-selenium-java</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>14</maven.compiler.source>
+        <maven.compiler.target>14</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api -->
```
Comment (github4ta) [inline pom.xml]: комментарии можно удалить

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.11.2</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
```
Comment (github4ta) [inline pom.xml]: комментарии можно удалить

File: pom.xml (hunk 1 above)
Comment (Mits1388) [reply pom.xml]: исправлено

File: pom.xml (hunk 2 above)
Comment (Mits1388) [reply pom.xml]: исправлено
